src/js/setUpSite.js

```diff
@@ -236,11 +238,31 @@ const setUpSite = async () => {
   setUpAbout();
 
   const map = createMap();
-  await Promise.all([setUpCitiesLayer(map), setUpParkingLotsLayer(map)]);
+  await Promise.all([setUpCitiesLayer(map), setUpParkingLotsLayer(map), setUpSearch(initialCityId, "atlanta-ga")]);
 
   // There have been some issues on Safari with the map only rendering the top 20%
   // on the first page load. This is meant to address that.
   map.invalidateSize();
 };
 
+const setUpSearch = (initialCityId, fallbackCityId) => {
+  const element = document.getElementById("city-choice"); 
+    var choices = new Choices(element, {
+      maxItemCount: 1,
+      placeholder: true,
+      placeholderValue: "City Search",
+      itemSelectText: "Select",
+      searchEnabled: true,
+      position: "bottom"
+  }); 
+
+  // Set up map to update when city selection changes.
+  const cityToggleElement = document.getElementById("city-choice");
+  cityToggleElement.addEventListener("change", () => {
+     const cityId = cityToggleElement.value;
+     setMapToCity(map, cityId, cities[cityId]);
+  });
```

Comment:
This event listener is already being set in `setUpCitiesLayer` so I think you can remove this. Also, there is no variable for `cities` so the console throws an error.

Reply:
@tunglinn You're totally right, I overlooked that part. Let me make that change!